package.json

```diff
@@ -1,64 +1,65 @@
 {
-  "name": "npx",
+  "name": "app-prototype",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/enzyme": "^3.10.3",
-    "@types/enzyme-adapter-react-16": "^1.0.5",
-    "@types/jest": "24.0.15",
-    "@types/node": "15.3.0",
-    "@types/react": "16.8.23",
-    "@types/react-dom": "16.8.4",
+    "@types/node": "^15.6.1",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "classnames": "^2.3.1",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-jest": "^22.6.4",
-    "eslint-plugin-prettier": "^3.1.0",
-    "husky": "^2.4.0",
-    "jest-enzyme": "^7.0.2",
-    "jest-junit": "^6.4.0",
-    "lint-staged": "^8.2.0",
-    "node-sass-chokidar": "^1.3.5",
-    "npm-run-all": "^4.1.3",
-    "prettier": "1.18.2",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.0.1",
+    "eslint": "^7.23.0",
+    "eslint-config-prettier": "^8.1.0",
```

Comment:
nonblocking:
should these (eslint + plugins) go to `devdependencies` too?

Reply:
Good question... I had tried to move these to `devDependencies`, but something hard-coded in `react-scripts build` requires these packages. If I skip installing dev dependencies by using `yarn --install`, then I get this failure:

```
 => [3/6] COPY package.json yarn.lock ./                                                                                                                  0.1s
 => [4/6] RUN npx yarn@1.12.1 install --production                                                                                                       50.9s
 => [5/6] COPY . .                                                                                                                                        0.2s 
 => ERROR [6/6] RUN npm run build                                                                                                                         9.3s 
------                                                                                                                                                         
 > [6/6] RUN npm run build:                                                                                                                                    
#10 0.674                                                                                                                                                      
#10 0.674 > app-prototype@0.1.0 build /app-prototype                                                                                                           
#10 0.674 > react-scripts build                                                                                                                                
#10 0.674                                                                                                                                                      
#10 2.459 Creating an optimized production build...
#10 9.143 Failed to compile.
#10 9.143 
#10 9.144 Failed to load config "prettier" to extend from.
#10 9.144 Referenced from: /app-prototype/.eslintrc.js
#10 9.144 
#10 9.144 
#10 9.188 npm ERR! code ELIFECYCLE
#10 9.188 npm ERR! errno 1
#10 9.191 npm ERR! app-prototype@0.1.0 build: `react-scripts build`
#10 9.191 npm ERR! Exit status 1
#10 9.191 npm ERR! 
#10 9.191 npm ERR! Failed at the app-prototype@0.1.0 build script.
#10 9.191 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
#10 9.197 
#10 9.197 npm ERR! A complete log of this run can be found in:
#10 9.197 npm ERR!     /root/.npm/_logs/2021-05-26T16_53_39_005Z-debug.log
------
executor failed running [/bin/sh -c npm run build]: exit code: 1
```

But! I just found this helpful comment: https://github.com/facebook/create-react-app/issues/10065#issuecomment-820439175

I will add `DISABLE_ESLINT_PLUGIN=true` as suggested in the comment when doing a production build. That fixes the issue.

Reply:
Addressed in c2fea32